src/service/webstorage.service.ts

```diff
@@ -68,14 +68,15 @@ export abstract class WebStorageService {
             filter((event: NgxStorageEvent) => {
                 if (!key) { return true; }
                 if (exactMatch) {
-                    if (key.startsWith(Config.prefix)) {
+                    if (!key.startsWith(Config.prefix)) {
```

Comment:
Why? It doesn't make sens to add Config.prefix to the key below if the key already starts with prefix...

Reply:
Tbh, I don't remember exactly why I did that... but I suppose, it is because `event.key` is a key without prefix, and that fix is to observe `key` without prefix, so, for example:
**Before:**
```ts
if (exactMatch) {
  if (key.startsWith(Config.prefix)) {
  // if ('test'.startsWith('im.')) { // false
    return event.key === key;
  }
  return event.key === Config.prefix + key;
  // return 'test' === 'im.' + 'test'; // false
}
```

**After:**
```ts
if (exactMatch) {
  if (!key.startsWith(Config.prefix)) {
  // if (!'test'.startsWith('im.')) { // true
    return event.key === key;
    // return 'test' === 'test'; // true
  }
  return event.key === Config.prefix + key;
}
```

But it won't work for `key` with prefix in both cases - Example 2:
**Before:**
```ts
if (exactMatch) {
  if (key.startsWith(Config.prefix)) {
  // if ('im.test'.startsWith('im.')) { // true
    return event.key === key;
    // return 'test' === 'im.test'; // false
  }
  return event.key === Config.prefix + key;
}
```

**After:**
```ts
if (exactMatch) {
  if (!key.startsWith(Config.prefix)) {
  // if (!'im.test'.startsWith('im.')) { // false
    return event.key === key;
  }
  return event.key === Config.prefix + key;
  // return 'test' === 'im.' + 'im.test'; // false
}
```

But I don't remember exactly. It was a while ago.